Multisig/UI/UI Library/TopTabView/TopTabChildView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TopTabChildView.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 14.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+// Wraps together label and content view to be used in the TopTabView
+struct TopTabChildView<Label: View, Content: View>: View {
+
+    var label: Label
```

Comment:
Do we need to have label?

Reply:
this is a content of the tab bar (icon + text)

Reply:
I can rename it to the `tabItemContent`

Reply:
or `tabButtonLabel`

Reply:
I took the name from the SwiftUI's tabItem:

```
    public func tabItem<V>(@ViewBuilder _ label: () -> V) -> some View where V : View
```

it declares the content as the `label`, so that's why I used `label`. It seems to be the preferred name for such things.

Reply:
keep it label please